default/scripting/buildings/GAS_GIANT_GEN.focs.txt

```diff
@@ -22,7 +22,8 @@ BuildingType
             ]
             stackinggroup = "GAS_GIANT_GEN_STACK"
             priority = [[VERY_LATE_PRIORITY]]
-            effects = SetTargetIndustry value = Value + 10
+            effects = SetTargetIndustry value = Value + 5 
```

Comment:
This might be better filtered by scope, rather than a complicated statistic in the "calculation" of 5 or 10.

Reply:
You mean duplicating the EffectGroups one for GasGiants and one for not GasGiants, right?

Reply:
yes